modules/local/linux/awk_script.nf

```diff
@@ -1,22 +1,11 @@
-include { initOptions; saveFiles; getSoftwareName; getProcessName } from '../common/functions'
-
-
-params.options = [:]
-options    = initOptions(params.options)
-
 process AWK_SCRIPT {
     tag "$meta.id"
-    label 'process_low'
-    publishDir "${params.outdir}",
-        mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+    label 'process_min'
 
     conda (params.enable_conda ? "conda-forge::gawk=5.1.0" : null)
-    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img"
-    } else {
-        container "biocontainers/biocontainers:v1.2.0_cv1"
-    }
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
+        'quay.io/biocontainers/gawk:5.1.0' }"
```

Comment:
Does moving to `quay.io/biocontainers/gawk:5.1.0` resolve the same issue that GCP had with biocontainers?

https://nfcore.slack.com/archives/CE8SSJV3N/p1649444888893609

I was able to get past the issue by replacing all `biocontainers/biocontainers:v1.2.0_cv1 ` containers with `ubuntu:20.04`. I don't know much about gawk, admittedly.

Reply:
Well spotted! 👀 Think it should be fine 👍🏽 Although, I suspect that the `ubuntu` container will have many more and up-to-date Linux commands so probably better to use that for any processes requiring that sort of functionality. Also saves having to pull multiple containers. An example [here](https://github.com/nf-core/modules/blob/f89b76afa451e1ecf2d585e9e7a41657d8d375d5/modules/cat/fastq/main.nf#L5-L9)

Reply:
PS: When you run the `biocontainers/biocontainers:v1.2.0_cv1` container you see that the default user is `biodocker` and that's the problem:

```
$ docker run -it biocontainers/biocontainers:v1.2.0_cv1 bash
biodocker@814ab53a89af:/data$
```

```
$ docker run -it ubuntu:20.04 bash
root@6f0c6df05b84:/#
```

Reply:
Ah I see

Reply:
Can i close this? I am a bit confused @drpatelh @dshinzie

Reply:
This should fix it https://github.com/nf-core/cutandrun/pull/128 @dshinzie can test `dev` to confirm after this is merged